client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -98,6 +100,7 @@ function ReaderSubscriptionListItem( {
 					className="reader-subscription-list-item__site-url"
 				>
 					{ formatUrlForDisplay( siteUrl ) }
+					{ feed && feed.last_update && ( ' ' + translate( 'updated' ) + ' ' + lastUpdatedDate ) }
```

Comment:
To ensure proper translation, I think we need to include the lastUpdatedDate in the translated string with `args`.